spring/src/main/java/io/seata/spring/annotation/GlobalTransactionalInterceptor.java

```diff
@@ -190,4 +237,37 @@ public void onChangeEvent(ConfigurationChangeEvent event) {
             disable = Boolean.parseBoolean(event.getNewValue().trim());
         }
     }
+
+    /**
+     * auto upgrade service detection
+     */
+    private static void startSelfCheck() {
+        ScheduledThreadPoolExecutor executor =
```

Comment:
I think ScheduledThreadPoolExecutor should be static and outside the method.

Reply:
received, processed,thx